source/write/bulk-write.txt

```diff
@@ -0,0 +1,334 @@
+
+To perform an insert operation, create an ``insert_one`` hash and specify
+the document you want to insert.
+
+The following example creates an ``insert_one`` hash:
+
+.. literalinclude:: /includes/write/bulk-write.rb
+   :start-after: start-bulk-insert-one
+   :end-before: end-bulk-insert-one
+   :language: ruby
+   :copyable:
+   :dedent:
+
+To insert multiple documents, create a separate ``insert_one`` hash
+for each document.
+
+.. important::
+
+   When performing a bulk operation, the ``insert_one`` operation cannot
+   insert a document with an ``_id`` that already exists in the
+   collection. In this situation, the driver throws a
+   ``MongoBulkWriteException``.
+
+Update Operations
+~~~~~~~~~~~~~~~~~
+
+To update a document, create an ``update_one`` hash and pass
+the following arguments:
+
+- **Query filter** that specifies the criteria used to match documents in your collection.
+- Update operation you want to perform. For more information about update
+  operations, see the :manual:`Field Update Operators
+  </reference/operator/update-field/>` guide in the {+mdb-server+} manual.
+
+The ``update_one`` operation specifies an update for *the first*
+document that matches your query filter.
+
+The following example creates an ``update_one`` hash:
+
+.. literalinclude:: /includes/write/bulk-write.rb
+   :start-after: start-bulk-update-one
+   :end-before: end-bulk-update-one
+   :language: ruby
+   :copyable:
+   :dedent:
+
+To update multiple documents, create an ``update_many`` hash and pass
+the same arguments as for an ``update_one`` operation. The ``update_many``
+operation specifies updates for *all* documents that match your query
+filter.
+
+The following example creates an ``update_many`` hash:
+
+.. literalinclude:: /includes/write/bulk-write.rb
+   :start-after: start-bulk-update-many
+   :end-before: end-bulk-update-many
+   :language: ruby
+   :copyable:
+   :dedent:
+
+Replace Operations
+~~~~~~~~~~~~~~~~~~
+
+A replace operation removes all fields and values of a specified document and
+replaces them with new fields and values that you specify. To perform a
+replace operation, create a ``replace_one`` hash and pass the following
+arguments:
+
+- Query filter that specifies the criteria used to match documents in your collection
+- Replacement document that specifies the new fields and values to insert
+
+The following example creates a ``replace_one`` hash:
+
+.. literalinclude:: /includes/write/bulk-write.rb
+   :start-after: start-bulk-replace-one
+   :end-before: end-bulk-replace-one
+   :language: ruby
+   :copyable:
+   :dedent:
+
+To replace multiple documents, you must create a
+``replace_one`` hash for each document.
+
+Delete Operations
+~~~~~~~~~~~~~~~~~
+
+To delete a document, create a ``delete_one`` hash and pass a
+query filter specifying the document you want to delete. The
+``delete_one`` operation provides instructions to delete
+only *the first* document that matches your query filter.
```

Comment:
The way this is worded makes it seem like there are separate instructions provided to delete only the first one document. 
```suggestion
``delete_one`` operation deletes
only *the first* document that matches your query filter.
```